src/main/kotlin/app/revanced/patches/ad/VideoAdsPatch.kt

```diff
@@ -0,0 +1,103 @@
+package app.revanced.patches.ad
+
+import app.revanced.patcher.PatcherData
+import app.revanced.patcher.extensions.AccessFlagExtensions.Companion.or
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.patch.*
+import app.revanced.patcher.signature.MethodMetadata
+import app.revanced.patcher.signature.MethodSignature
+import app.revanced.patcher.signature.MethodSignatureMetadata
+import app.revanced.patcher.signature.PatternScanMethod
+import app.revanced.patcher.smali.asInstructions
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+private val compatiblePackages = arrayOf("com.google.android.youtube")
```

Comment:
Maybe put this is in a `Shared.kt` instead and make it `internal`?

Reply:
As suggested in https://github.com/ReVancedTeam/revanced-patches/pull/2#discussion_r850279985, this parameter should be unique to each patch. It will be used to determine which patch should be allowed for what package.